conf.yaml

```diff
@@ -2092,6 +2072,26 @@ contents:
                 repo:   elasticsearch
                 path:   docs/Versions.asciidoc
                 exclude_branches:   [ 2.4, 2.3, 2.2, 2.1, 2.0, 1.7, 1.6, 1.5, 1.4, 1.3, 0.90 ]
+          - title:      Legacy Topbeat Reference
```

Comment:
Sorry, one change I missed. Let's remove "Legacy" from this title.

Reply:
Good find